server/studyspaces.py

```diff
@@ -9,37 +9,16 @@
 from penn.base import APIError
 from .models import StudySpacesBooking, User
 from .penndata import studyspaces
+from .penndata import wharton
 from .base import cached_route
 
+def get_sessionid():
+    return "l87ugy7ha9fabhxil71zwshbsl9ogl7x"
```

Comment:
Don't put any sensitive variables in source code. The repository is public, so everyone can see them.

Instead, you want to use `os.environ` and read the sensitive value from an environment variable. You can then later set this environment variable on the server when you deploy to production.